src/track/serato/markers.cpp

```diff
@@ -589,12 +592,16 @@ QList<CueInfo> SeratoMarkers::getCues() const {
         }
         case SeratoMarkersEntry::TypeId::Loop: {
             if (pEntry->hasStartPosition()) {
+                if (!pEntry->hasEndPosition()) {
+                    qWarning() << "SeratoMarkers: Loop" << loopIndex << "has no end position!";
```

Comment:
Usually this can't happen unless Serato is buggy or the Metadata is broken or has been edited externally. But we should never trust user data ;-)

I'll add a comment.